pkg/version/version_parser_test.go

```diff
@@ -360,3 +360,54 @@ func TestLess(t *testing.T) {
 		})
 	}
 }
+
+func TestPreviousMinor(t *testing.T) {
```

Comment:
The `GetPrevMinor` method was discarding the prerelease and build metadata information.